daemons/dss-checkout-sfn/app.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss.util.state_machine.checkout_states import state_machine_def
+from dss.util.email import send_checkout_success_email, send_checkout_failure_email
+from dss.util.checkout import (parallel_copy, get_dst_bundle_prefix, get_manifest_files,
+                               validate_file_dst, validate)
+
+app = domovoi.Domovoi()
```

Comment:
L15-22 seems like a jumble of initialization steps (which is probably fine for now).  It might be nice to kind of group them by:

1. setting up globals:
```
app = domovoi.Domovoid()
logger = dss.get_logger()
```
2. Config-related stuff:
```
dss.Config.set_config(dss.BucketConfig.NORMAL)
email_sender = dss.Config.get_notification_email()
default_checkout_bucket = dss.Config.get_s3_checkout_bucket()
```

Reply:
Re-ordered